addon/index.ts

```diff
@@ -4,8 +4,8 @@ export {
   getWaiters,
   reset,
```

Comment:
Should probably be underscored, right?

Reply:
Ya, I wanted to go through the process of using these things before I determined, once and for all, what was private vs. public. I think we're close to determining this now.

Reply:
Done.